src/rgw/rgw_common.cc

```diff
@@ -1083,6 +1083,7 @@ bool verify_user_permission(const DoutPrefixProvider* dpp,
                             const uint64_t op)
 {
   auto usr_policy_res = eval_user_policies(user_policies, s->env, boost::none, op, res);
+  ldpp_dout(s, 0) << "User Policy result is: " << static_cast<int>(usr_policy_res) << dendl;
```

Comment:
If these are intended to stick around, they should probably be 5 instead of 0. (There was a post on ceph-users recently where someone got a surprising performance boost by dialing the debugging way down, and we don't want to make it too difficult to do.)

If they're just temporary, they're fine how they are.

Reply:
they shouldn't be 0, I'd say for a print like this to say, it would need to be 15 or higher

Reply:
Yes Matt, this is part of a debug commit, which I intend to remove later